### PR TITLE
Add Webhook Credential Hashes spec

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

**Comment by slavcodev** [comment]
wonder what is better endpoint `/credential-hashes/webhook/` or `/credential-hashes/webhooks/` , thoughts?

**Comment by adamaltman** [comment]
I think with `/webhooks/`.

File: spec/definitions/WebhookCredentials/WebhookAuthorization.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+type: object
+discriminator: type
+properties:
+  token:
+    readOnly: true
+    allOf:
+      - $ref: "#/definitions/ResourceId"
+  host:
```
Comment (adamaltman) [inline spec/definitions/WebhookCredentials/WebhookAuthorization.yaml]: is there no port too?  Is host just the base domain, or the full url?  Example:

1. https://example.com
2. example.com
3. https://example.com/receive/webhook/here

Comment (slavcodev) [inline spec/definitions/WebhookCredentials/WebhookAuthorization.yaml]: It's host (no scheme, no path), just to link credential by host, the port also no need here. 
To be clear, it's not webhook URL. The domain from webhook URL will be compare with this host before allow to use credential.

Comment (adamaltman) [inline spec/definitions/WebhookCredentials/WebhookAuthorization.yaml]: Okay, so it's like 2. `example.com`.  The logic makes sense.

Comment (slavcodev) [inline spec/definitions/WebhookCredentials/WebhookAuthorization.yaml]: Yes. Only domain/host `example.com`. May be some description and example would be helpful.

File: spec/paths/credential-hashes@webhooks@{token}.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+parameters:
+  -
+    name: token
```
Comment (slavcodev) [inline spec/paths/credential-hashes@webhooks@{token}.yaml]: `hash` ?

Comment (vova07) [inline spec/paths/credential-hashes@webhooks@{token}.yaml]: fixed!

**Pull request merged**